site/src/components/MoreMenu/MoreMenu.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import GrassIcon from "@mui/icons-material/Grass";
+import KitesurfingIcon from "@mui/icons-material/Kitesurfing";
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+  MoreMenu,
+  MoreMenuContent,
+  MoreMenuItem,
+  MoreMenuTrigger,
+  ThreeDotsButton,
+} from "./MoreMenu";
+import { action } from "@storybook/addon-actions";
+import { expect, screen, userEvent, waitFor, within } from "@storybook/test";
+
+const meta: Meta<typeof MoreMenu> = {
+  title: "components/MoreMenu",
+  component: MoreMenu,
+};
+
+export default meta;
+type Story = StoryObj<typeof MoreMenu>;
+
+const Example: Story = {
+  args: {
+    children: (
+      <>
+        <MoreMenuTrigger>
+          <ThreeDotsButton />
+        </MoreMenuTrigger>
+        <MoreMenuContent>
+          <MoreMenuItem onClick={() => action("grass")}>
+            <GrassIcon />
+            Touch grass
+          </MoreMenuItem>
+          <MoreMenuItem onClick={() => action("water")}>
+            <KitesurfingIcon />
+            Touch water
+          </MoreMenuItem>
+        </MoreMenuContent>
+      </>
+    ),
+  },
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
+
+    await step("Open menu", async () => {
+      await userEvent.click(canvas.getByRole("button"));
```

Comment:
To be clear, not blocking on this – it's more a note for myself for the accessibility backlog I'm slowly putting together:

It'd be nice if we could swap a selector like this in without things breaking:
```tsx
// Important thing is having any kind of accessible name - word choice doesn't matter
const menuButton = canvas.getByRole("button", {
  name: "Show options"
});
await userEvent.click(menuButton);
```

Reply:
I didn't even know that was a thing you could do! the testing library API is kind of just weird and mystic to me 😅